### PR TITLE
fix: add prechangelog script if does not exist

### PR DESCRIPTION
**Pull request by danzrou**

### CODE REVIEW

File: src/init/initBotConfiguration.ts [hunk 1]
```diff
@@ -53,11 +53,17 @@ function createVersionFile(file: '.versionrc.json' | '.versionrc.js', webHookLin
...
-  const prechangelog = config.scripts?.prechangelog;
-  if (prechangelog) {
-    config.scripts.prechangelog = `${prechangelog} && npm run version-bot:build-message`;
-  }
+  const script = `npm run version-bot:build-message`;
+  const current = config.scripts?.prechangelog;
+  const updated = (current && `${current} && ${script}`) || `${script}`;
+
+  config.scripts = {
+    ...config.scripts,
```
Comment (shaharkazaz) [inline src/init/initBotConfiguration.ts:61]: We need to protect in the case where `config.scripts` is nullish?

Comment (danzrou) [reply src/init/initBotConfiguration.ts:61]: In case its nullish it will only have prechangelog - nothing breaks here

Comment (shaharkazaz) [reply src/init/initBotConfiguration.ts:61]: you are using spread, `...undefined` will throw

Comment (danzrou) [reply src/init/initBotConfiguration.ts:61]: Am I missing something? 
https://stackblitz.com/edit/typescript-riaocm

Comment (shaharkazaz) [reply src/init/initBotConfiguration.ts:61]: Nope, I guess I did something on my end 🤔

**Pull request merged**